core/src/main/scala/KeyRange.scala

```diff
@@ -0,0 +1,7 @@
+package kr.ac.postech.paranode.core
+
+case class KeyRange(from: Key, to: Key) {
+  def includes(key: Key): Boolean = (from <= key) && (key < to)
```

Comment:
I wonder if it is acceptable that the final end of the KeyRange is not included in the partition. If the master sets the lowest key to the very end of the KeyRange, these records will be omitted from every partition.

I mean, below case, we cannot include 0x9 key in partitions

```scala
val firstKeyRange = KeyRange(new Key(Array(0x0)), new Key(Array(0x3)))
val secondKeyRange = KeyRange(new Key(Array(0x3)), new Key(Array(0x6)))
val thirdKeyRange = KeyRange(new Key(Array(0x6)), new Key(Array(0x9)))
```

Reply:
Regarding `https://github.com/betarixm/434project/pull/58#discussion_r1390790192`, I want to not include the end within range. However, your comment make sense. Maybe it is okay to include end of key.

Reply:
778f9fb06c45faf6efcaad42f30a44dc22df31e6